srcpkgs/tor/template

```diff
@@ -14,7 +14,7 @@ license="BSD-3-Clause"
 homepage="https://www.torproject.org/"
 changelog="https://gitweb.torproject.org/tor.git/plain/ReleaseNotes"
```

Comment:
changelog URL is broken now, could you fix it? https://gitlab.torproject.org/tpo/core/tor/-/raw/main/ChangeLog

Reply:
ok done (even though the gitweb url is actually still working for me)